libarchive/test/test_write_disk_secure.c

```diff
@@ -178,6 +178,26 @@ DEFINE_TEST(test_write_disk_secure)
 	assert(S_ISDIR(st.st_mode));
 	archive_entry_free(ae);
 
+	/*
+	 * Without security checks, we should be able to
+	 * extract an absolute path.
+	 */
+	assert((ae = archive_entry_new()) != NULL);
+	archive_entry_copy_pathname(ae, "/tmp/abs");
+	archive_entry_set_mode(ae, S_IFREG | 0777);
+	assert(0 == archive_write_header(a, ae));
+	assert(0 == archive_write_finish_entry(a));
```

Comment:
You should include a check here to verify that /tmp/abs was actually created in this case.  I think the following is sufficient:

assert_file_exists("/tmp/abs")